tests/MongoDB.Driver.Tests/Specifications/command-monitoring/BulkWriteTest.cs

```diff
@@ -48,32 +49,53 @@ protected override bool TrySetArgument(string name, BsonValue value)
                 case "requests":
                     _requests = ParseRequests((BsonArray)value).ToList();
                     return true;
+                case "options":
+                    _options = ParseOptions(value.AsBsonDocument);
```

Comment:
Lines 53 and 56 seem to be in conflict.

If `ordered` occurs before `options` then line 53 will overwrite whatever line 56 did.

Reply:
Do lines 55-57 just need to be removed?